esp/src/src/ESPRequest.ts

```diff
@@ -134,7 +134,7 @@ class RequestHelper {
     send(service, action, params?): Promise<any> {
         if (!this.isSessionCall(service, action) && (!this.hasServerSetCookie() || (this.hasAuthentication() && !this.isAuthenticated()))) {
             // @ts-ignore
-            window.location.reload(true);
+            // window.location.reload(true);
```

Comment:
commented code

Reply:
I wasn't sure why that reload was there, and removing hasn't seemed to have any effect. But I'd just left it there for the time being